src/core/server/http/http_server.test.ts

```diff
@@ -139,7 +143,7 @@ test('log listening address after started when configured with BasePath and rewr
 });
 
 test('valid params', async () => {
-  const router = new Router('/foo', logger, enhanceWithContext);
+  const router = new Router('/foo', logger, enhanceWithContext, config$);
```

Comment:
Now may be a good time to switch to named arguments?

Reply:
+1 for that.

Reply:
Ended up not needing to change the constructor, so I did not do this.